tests/test_api_urls.py

```diff
@@ -71,6 +71,19 @@ def test_api_labels_without_auth(self):
         self.assertEqual(rv.status_code, 200)
         self.assertEqual(rv.content_type, 'application/json')
 
+    def test_api_comments_link_header_auth(self):
+        '''API access to comments greater than 30 returns pagination in Link header of the response.'''
+        query_string = {'callback': 'foo'}
```

Comment:
Cool! I'd recommend just leaving a comment in the test, something like "Force a JSONP callback response because it gives us the headers we want to test."